### PR TITLE
Prebid Core: Configurable maxbid

### PR DESCRIPTION
**Pull request by mkomorski**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter  
- [ ] Updated bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Other information
#11252

### CODE REVIEW

File: src/auction.js [hunk 1]
```diff
@@ -974,3 +977,19 @@ function groupByPlacement(bidsByPlacement, bid) {
   bidsByPlacement[bid.adUnitCode].bids.push(bid);
   return bidsByPlacement;
 }
+
+/**
+ * isValidPrice is price validation function 
+ * which checks if price from bid response
+ * is not higher than top limit set in config
```
Comment (ChrisHuie) [inline src/auction.js:984]: Can you convert this comment to jsdoc syntax so an ide can pick up for syntax highlighting

Comment (mkomorski) [reply src/auction.js:984]: done

File: test/spec/auctionmanager_spec.js [hunk 2]
```diff
@@ -1467,6 +1470,19 @@ describe('auctionmanager.js', function () {
       config.getConfig.restore();
       store.store.restore();
     });
+
+    it('should reject bid for price higher than limit for the same currency', () => {
+      sinon.stub(auction, 'addBidRejected');
+      setCurrencyConfig({ adServerCurrency: 'USD' })
```
[the remark this answers is not in the document]
Comment (mkomorski) [reply test/spec/auctionmanager_spec.js:1476]: `setCurrencyConfig` is invoked here in order to run initCurrency() from currency module. There is `addBidResponse` hook binding inside, so without it currency conversion won't run I believe. maxBid is a value, did you mean 'maxBidCur'? If so I agree :)

[the remark this answers is not in the document]
Comment (mkomorski) [reply test/spec/auctionmanager_spec.js]: Did you mean `maxBidCur`? 👍

**Review by dgirardi — APPROVED** [review]

**Pull request merged**